testsuite/ada_lsp/range_formatting/W103-015_GitHub-1086/test.json

```diff
@@ -0,0 +1,330 @@
+                                        14,
+                                        15,
+                                        16,
+                                        17,
+                                        18,
+                                        19,
+                                        20,
+                                        21,
+                                        22,
+                                        23,
+                                        24,
+                                        25,
+                                        26
+                                    ]
+                                }
+                            },
+                            "documentHighlight": {
+                                "dynamicRegistration": false
+                            },
+                            "codeAction": {
+                                "dynamicRegistration": false,
+                                "codeActionLiteralSupport": {
+                                    "codeActionKind": {
+                                        "valueSet": [
+                                            "quickfix",
+                                            "refactor",
+                                            "refactor.extract",
+                                            "refactor.inline",
+                                            "refactor.rewrite",
+                                            "source",
+                                            "source.organizeImports"
+                                        ]
+                                    }
+                                },
+                                "isPreferredSupport": true
+                            },
+                            "formatting": {
+                                "dynamicRegistration": false
+                            },
+                            "rangeFormatting": {
+                                "dynamicRegistration": false
+                            },
+                            "rename": {
+                                "dynamicRegistration": false
+                            },
+                            "publishDiagnostics": {
+                                "relatedInformation": false,
+                                "codeDescriptionSupport": false,
+                                "tagSupport": {
+                                    "valueSet": [
+                                        1,
+                                        2
+                                    ]
+                                }
+                            },
+                            "semanticTokens": {
+                                "dynamicRegistration": false,
+                                "requests": {
+                                    "full": true,
+                                    "range": true
+                                },
+                                "tokenTypes": [
```

Comment:
Can't we avoid all the settings? Might be tricky to edit the tests if these things change one day

Reply:
Agree. Only the "processId" and "rootUri" are needed.